src/redux/settings.ts

```diff
@@ -41,7 +42,7 @@ const SETTINGS_UPDATE_ACCOUNT_SETTINGS_SUCCESS = 'settings/SETTINGS_UPDATE_ACCOU
  */
 interface SettingsState {
   appIcon: string;
-  accountAddress: string;
+  accountAddress: Address;
```

Comment:
this might be problematic in other areas of the app and tslint might complain to you about it